TrackingTools/PatternTools/plugins/TSCBLBuilderNoMaterialESProducer.cc

```diff
@@ -74,10 +74,10 @@ TSCBLBuilderNoMaterialESProducer::ReturnType
 TSCBLBuilderNoMaterialESProducer::produce(const TrackingComponentsRecord& iRecord)
 {
    using namespace edm::es;
-   TSCBLBuilderNoMaterialESProducer::ReturnType pTSCBLBuilderNoMaterial(new TSCBLBuilderNoMaterial()) ;
+   auto pTSCBLBuilderNoMaterial = std::make_unique<TSCBLBuilderNoMaterial>() ;
 
 
-   return pTSCBLBuilderNoMaterial ;
+   return std::move(pTSCBLBuilderNoMaterial) ;
```

Comment:
The `std::move` before the return is not needed (and actually can turn off a compiler optimization).